### PR TITLE
Modified 2 files

### PR DESCRIPTION
**Pull request by nimrodmatan**
Committed by ops@velocitycareerlabs.com on behalf of nimrod.matan@velocitycareerlabs.com

### CODE REVIEW

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 1]
```diff
@@ -714,6 +731,83 @@
         "id": "vidm1pqmlp4od"
       }
     },
+"CompactPrivileges ": {
```
Comment (sloops77) [inline Layer 2/NCSBNMultiLicenseV1.0.json:734]: add more spaces to align the text under the `}` properly

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 2]
```diff
@@ -714,6 +731,83 @@
...
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "Status": {
+          "type": "string",
+          "enum": [
+          "Active",
```
Comment (sloops77) [inline Layer 2/NCSBNMultiLicenseV1.0.json:788]: "Active", Expired and Terminated and `]` needs to be indented by two spaces

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 3]
```diff
@@ -714,6 +731,83 @@
...
+        },
+        "RenewalDate": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "Status": {
+          "type": "string",
```
Comment (sloops77) [inline Layer 2/NCSBNMultiLicenseV1.0.json:786]: probably should permit Null. check with them

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 4]
```diff
@@ -714,6 +731,83 @@
...
+      "required": [
+        "JurisdictionAbbreviation",
+        "Jurisdiction",
+        "HomeJurisdictionAbbreviation",
+        "HomeJurisdiction",
+        "IssueDate",
+        "ExpirationDate",
+       "RenewalDate",
+       "Status",
+       "Number "
```
Comment (sloops77) [inline Layer 2/NCSBNMultiLicenseV1.0.json:808]: remove the trailing space after Number
pls align RenewalDate, Status, Number

**Review by sloops77 — CHANGES_REQUESTED** [review]

File: Layer 2/MultiLicenseV1.0.json [hunk 5]
```diff
@@ -612,6 +619,15 @@
...
         },
+        "CompactPrivileges": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "$ref": "#/definitions/CompactPrivileges"
+          }
+          },
```
Comment (sloops77) [inline Layer 2/MultiLicenseV1.0.json:630]: reduce indent

File: Layer 2/MultiLicenseV1.0.json [hunk 6]
```diff
@@ -714,6 +731,86 @@
...
+            "Expired",
+            "Terminated"
+          ]
+        },
+        "Number": {
+          "type": [
+            "string",
+            "null"
+          ]
+      },
```
Comment (sloops77) [inline Layer 2/MultiLicenseV1.0.json:801]: looks like a closing } is missing here

File: Layer 2/MultiLicenseV1.0.json [hunk 7]
```diff
@@ -897,10 +985,10 @@
...
     "Name",
-    "NcsbnId",
+    "Id",
     "Messages",
     "Licenses",
-    "RNAuthorizationsToPractice",
-    "PNAuthorizationsToPractice"
+    "AuthorizationToPractice"
   ]
+}
```
Comment (sloops77) [inline Layer 2/MultiLicenseV1.0.json:993]: should be removed

**Review by sloops77 — CHANGES_REQUESTED** [review]

File: Layer 2/MultiLicenseV1.0.json (hunk 6 above)
Comment (nimrodmatan) [reply Layer 2/MultiLicenseV1.0.json:801]: I don't think so. And stoplight's validation does not allow me to end one. The closing } is in row 813.

File: Layer 2/MultiLicenseV1.0.json (hunk 7 above)
Comment (nimrodmatan) [reply Layer 2/MultiLicenseV1.0.json:993]: Stoplight's validation forces me to keep it,.

Comment (sloops77) [reply Layer 2/MultiLicenseV1.0.json:993]: did you try both removing here and adding above at the same time?

Comment (nimrodmatan) [reply Layer 2/MultiLicenseV1.0.json:993]: I'm using stoplight as the editor and was relying on its validation. isn't it good enough?

Comment (sloops77) [reply Layer 2/MultiLicenseV1.0.json:993]: no - because the document isn't actually correct and the required props aren't being managed properly

Comment (nimrodmatan) [reply Layer 2/MultiLicenseV1.0.json:993]: which tool should I use then?

**Review by sloops77 — APPROVED** [review]

**Pull request merged**